### PR TITLE
tests/br: fix bug in issue #1158

### PR DESCRIPTION
**Pull request by Leavrth**
### What problem does this PR solve? 

fix bug in issue #1158

### What is changed and how it works?

for #1158 add sleep 1 to ensure tikv start up completely before restore

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note

 -No release note

### CODE REVIEW

**Comment by YuJuncen** [comment]
Maybe the utility function `ensure_tikv` would be suitable for this? (Eh, maybe needing some modify, say, ensure every TiKV nodes are up via interfaces of PD.)

https://github.com/pingcap/br/blob/701909b29ffa5b964d437e15fa9acfbf046255cb/tests/_utils/run_services#L144-L154

BTW, according to the log provided in #1158, seems TiKV 3 started after 5 seconds since restoration starting. I guess maybe 1s of sleeping isn't enough?

**Comment by Leavrth** [comment]
> Maybe the utility function `ensure_tikv` would be suitable for this? (Eh, maybe needing some modify, say, ensure every TiKV nodes are up and connected via interfaces of PD.)

Got it, I will modify the code and have a test.

>  Well, seems there is clock drifting between the host and the container.

Yes, there seems 2 different clocks in bash and in BR

**Review by 3pointer — APPROVED** [review]

File: tests/br_tikv_outage/run.sh [hunk 1]
```diff
@@ -71,6 +71,8 @@ github.com/pingcap/br/pkg/conn/hint-get-backup-client=1*return(\"$hint_get_backu
     backup_pid=$!
     single_point_fault $failure
     wait $backup_pid
+    # let tikv start up completely if backup is finished before tikv restarts
+    ensure_tikv
```
Comment (glorv) [inline tests/br_tikv_outage/run.sh:75]: shall we move this line into the case block to make it clearer which failures depend on this

Comment (Leavrth) [reply tests/br_tikv_outage/run.sh]: I add the line into kv_outage function where tikv restart. It seems be better.

**Review by glorv — APPROVED** [review]

**Review by 3pointer — APPROVED** [review]

**Pull request merged**